pkg/apis/certmanager/v1alpha1/types_issuer.go

```diff
@@ -291,6 +291,7 @@ type ACMEChallengeSolverHTTP01Ingress struct {
 
 type ACMEChallengeSolverHTTP01IngressPodTemplate struct {
 	metav1.ObjectMeta `json:"metadata,omitempty"`
+	corev1.PodSpec    `json:"spec,omitempty"`
```

Comment:
(note to self: create another structure here with only Tolerations and NodeSelector)